docs/getting-started/installation/helm.md

```diff
@@ -0,0 +1,353 @@
+# Helm
+
+<img src="https://avatars.githubusercontent.com/u/153325344?s=200&v=4" align="right" width="92" alt="homarr logo">
+
+![Version: 0.0.1](https://img.shields.io/badge/Version-0.0.1-informational?style=flat)
+![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat)
+![AppVersion: 0.0.1](https://img.shields.io/badge/AppVersion-0.0.1-informational?style=flat)
+
+A Helm chart to deploy homarr for Kubernetes
+
+**Homepage:** <https://homarr-labs.github.io/charts/charts/homarr/>
+
+## Source Code
+
+* <https://github.com/homarr-labs/homarr>
+
+## 📄 Requirements
+
+Kubernetes: `>=1.22.0-0`
+
+## 🍱 Dependencies
+
+| Repository | Name | Version |
+|------------|------|---------|
+| <https://charts.bitnami.com/bitnami> | mysql | 11.1.17 |
```

Comment:
Can you format this better?

Reply:
@manuel-rw I just reformated the table and i rebased the branch